articles/github-actions/index.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: github-actions
+title: Introduction to GitHub Actions
+description: GitHub Actions is an API for cause and effect on GitHub: orchestrate any workflow, based on any event, while GitHub manages the execution, provides rich feedback, and secures every step along the way.
+author: rohan-reddy
+date: 2020-07-08T00:00:00-07:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/github-actions/hero.jpg
+    alt: shell scripting example image
+---
+[Github Actions](https://github.com/features/actions) enables you to create custom and automated software development life cycle ([SDLC](https://en.wikipedia.org/wiki/Systems_development_life_cycle)) workflows directly in your [GitHub repository](https://github.com/github).
```

Comment:
I would probably say it allows you to create custom Continuous Integration (CI) flows in your repository rather than SDLC workflows. SDLC includes things like planning, analysis, and design, which as far as I'm aware github actions don't help with.